cmd/configuration/configuration.go

```diff
@@ -0,0 +1,108 @@
+package configuration
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/flyteorg/flytestdlib/logger"
+
+	"github.com/flyteorg/flytectl/pkg/configutil"
+
+	initConfig "github.com/flyteorg/flytectl/cmd/config/subcommand/config"
+	cmdcore "github.com/flyteorg/flytectl/cmd/core"
+	cmdUtil "github.com/flyteorg/flytectl/pkg/commandutils"
+	"github.com/flyteorg/flytestdlib/config/viper"
+	"github.com/manifoldco/promptui"
+
+	"github.com/spf13/cobra"
+)
+
+// Long descriptions are whitespace sensitive when generating docs using sphinx.
+const (
+	initCmdShort = `Generates flytectl config file in the user's home directory.`
+	initCmdLong  = `Creates a flytectl config file in flyte directory i.e ~/.flyte
+	
+Generate sandbox config. Flyte Sandbox is a fully standalone minimal environment for running Flyte. Read more about sandbox https://docs.flyte.org/en/latest/deployment/sandbox.html
+
+::
+
+ bin/flytectl configuration config 
+
+Generate remote cluster config. Read more about the remote deployment https://docs.flyte.org/en/latest/deployment/index.html
+	
+::
+
+ bin/flytectl configuration config --host=flyte.myexample.com
+	
+Generate flytectl config with a specific storage type
+::
+
+ bin/flytectl configuration config --host=flyte.myexample.com --storage=gcs
```

Comment:
instead of having him select the storage here and also in prompt , i would prefer to do it only once to avoid user providing gcs in command line args and then in prompt selecting something else
We can keep this flag as boolean and have all type of storage being available in the prompt. 

Over here we can add an example with how the prompt would show up in aws or gcs case.

Reply:
it will only prompt if user didn't pass flag for storage